visualization/package.json

```diff
@@ -70,8 +65,6 @@
 	"dependencies": {
 		"@angular/animations": "^15.1.1",
 		"@angular/cdk": "^15.1.1",
-		"@angular/common": "^15.1.1",
-		"@angular/core": "^15.1.1",
```

Comment:
We don't need them anymore?

Reply:
Good question, I'm not sure why they got removed. I will look this.